src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -146,8 +146,9 @@ interface MongoLayerMapping {
   lookaheadMaxSearchDistance: number
 }
 
-export interface MongoMedia {
+export interface MongoMedia extends MongoId {
   mediaId: string
+  mediaPath: string
```

Comment:
What is this for? As far as I can see this isn't used anywhere?

Reply:
A leftover that i'm glad you caught!